packages/analytics-js/src/components/utilities/loadOptions.ts

```diff
@@ -106,6 +102,9 @@ const normalizeLoadOptions = (
   if (!isNumber(normalizedLoadOpts.dataPlaneEventsBufferTimeout)) {
     delete normalizedLoadOpts.dataPlaneEventsBufferTimeout;
   }
+  if (!isObjectLiteralAndNotNull(normalizedLoadOpts.storage?.cookie)) {
```

Comment:
Also, remove undefined and null values.

Reply:
`isObjectLiteralAndNotNull` this function already removes undefined and null values.

Reply:
No, it merely checks for the presence of them.
In the else block, you should invoke the utility method `removeUndefinedAndNullValues`.